examples/example-phx-1_2/priv/repo/migrations/20180326205407_change_and_convert_page_data_type_in_backups_table.exs

```diff
@@ -0,0 +1,58 @@
+defmodule Example.Repo.Migrations.ChangeAndConvertPageDataTypeInBackupsTable do
```

Comment:
This is the migration that handles converting the binary compressed data to a simple json string (which is what the data is decompressed).